acceptance/tests/custom_facts/long_stderr_with_time_limit.rb

```diff
@@ -0,0 +1,155 @@
+test_name "Facter::Core::Execution doesn't kill process with long stderr message" do
+  tag 'risk:high'
+
+  file_content = <<-EOM
```

Comment:
You can make use of ruby here, to generate the needed text, eg:
```ruby
length = 1000
file_content = <<-EOM
   #!/bin/sh
   long_output=#{"Lorem ipsum dolor sit amet" * length }
   echo 'newfact=value_of_fact'
   1>&2 echo $long_output
   exit 1
EOM
```

And play with the `length` to achieve the expected output.

Also i think would be useful to add some description to this scenario and what is this test doing